### PR TITLE
Add new method Pipeline::writeStr

### PR DESCRIPTION
**Pull request by m-holger**
For performance

### CODE REVIEW

File: libqpdf/QPDFWriter.cc
Comment (jberkenbilt) [inline libqpdf/QPDFWriter.cc:984]: Does it make sense for this to take `std::string_view` too?

**Review by jberkenbilt — APPROVED** [review]
I'm fine with this change but will wait for your response before merging.

File: libqpdf/QPDFWriter.cc
Comment (jberkenbilt) [reply libqpdf/QPDFWriter.cc:984]: There's a case to be made for having part of qpdf 12 be to replace `std::string const&` with `std::string_view` in as many places as possible and to live with the slight performance penalty in the meantime to avoid lots of API clutter.

Comment (m-holger) [reply libqpdf/QPDFWriter.cc]: > Does it make sense for this to take std::string_view too?

Agree. Updated

> There's a case to be made for having part of qpdf 12 be to replace std::string const& with std::string_view 

Agree

**Comment by m-holger** [comment]
Superseded by decision re qpdf 12.

**Pull request closed without merging**
